widgets/dialogs.pas

```diff
@@ -296,6 +332,31 @@ constructor TMessageDialog.Create(AOwner: TComponent);
     finally
       FButtonPanel.EndUpdate;
     end;
+
+    fTitlePanel := TWPanel.Create(Self);
+    fTitlePanel.BeginUpdate;
+    try
+      fTitlePanel.Parent := Self;
+      fTitlePanel.BorderSpacing.Around := CControlsSpacing;
+      fTitlePanel.Height := CTitileHeight;
+      fTitlePanel.BevelOuter := bvNone;
+      fTitlePanel.Align := alTop;
+      fTitlePanel.Color:= clSkyBlue;
```

Comment:
Missing space before the assignment operator

Reply:
Fixed.